src/qaqc/download_and_flag_fxns/combine_hist_inc_data.R

```diff
@@ -29,28 +17,22 @@ combine_hist_inc_data <- function(incoming_data_list, historical_data_list) {
                           mutate(historical = TRUE,
                                  # ensure last_site_visit column has proper DT:
                                  last_site_visit = force_tz(last_site_visit, tzone = "MST"))
-                          # rename(cleaner_flag_old = cleaner_flag,
-                          #        over_50_percent_fail_window_old = over_50_percent_fail_window,
-                          #        flag_old = flag)
                       })
 
-  # bind all_data_summary and last_24_hours together
+  # bind summarized_incoming_data and last_24_hours together
   combined_hist_inc_data <- map(matching_indexes, function(index) {
     last_24_hours[[index]] %>%
-      select(-c(t_mean01,
+      select(-c(t_mean01, # why is this being removed? -jd
                 t_mean99,
                 t_slope_behind_01,
                 t_slope_behind_99,
                 t_sd_0199)) %>%
-      bind_rows(., mutate(incoming_data_list[[index]], historical = FALSE)[-1,]) %>%
+      bind_rows(., mutate(incoming_data_list[[index]], historical = FALSE)[-1,]) %>% # why are we removing the first row? -jd # there is a datetime issue on this bind
```

Comment:
@juandlt-csu is this because, if we are using the max start DT identified in the historical data, that max DT's data will also be pulled in the new api download. We don't want redundant DTs...

Reply:
Is this in regard to the row removal or the date time bind issue? I don't think that there should be an issue with redundant DTs given how we are re-appending the data to the historical data (we are lopping off the old subset and replacing it with the new subset+incoming flagged data)